fix(curriculum): description change of learn-css-variables-by-building-a-city-skyline (#47796)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-css-variables-by-building-a-city-skyline/5d822fd413a79914d39e98d5.md

# --description--

> Try to center your `.bb1` element, using these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`.

Reviewer: ```suggestion
Center your `.bb1` element by using these style properties: `display: flex;`, `flex-direction: column;`, and `align-items: center;`.
```
